### PR TITLE
Feature/onboarding modal

### PR DESCRIPTION
**Pull request by cfviotti**
This PR creates the feature of opening a modal specific for the onboardings.

The opinion onboarding should open when the logged user first opens a project. It should never appear again after clicking its button.

The text selection onboarding should open after passing through all available opinions and never appear again after clicking its button.

### CODE REVIEW

File: src/static/scss/components/_app-modal.scss [hunk 1]
```diff
@@ -134,9 +158,15 @@
   & > .content > .solid-button {
     font-size: $font-size-deci;
 
+    display: inline-flex;
+
     margin-top: 2rem;
-    margin-left: auto;
     padding: .5rem 2rem;
+
+    animation: fade-in-bottom 1s 4s $swift-out forwards;
```
Comment (Pettalves) [inline src/static/scss/components/_app-modal.scss]: I would take the animation and the delay out of the user experience.

File: src/static/scss/components/_app-modal.scss [hunk 2]
```diff
@@ -98,15 +113,24 @@
 
       text-align: left;
     }
+
+    @at-root .-onboarding#{&} {
+      overflow-y: unset;
+
+      max-width: 55rem;
```
Comment (Pettalves) [inline src/static/scss/components/_app-modal.scss:138]: Maybe it's better to follow something like the document page (document-body.scss): max-width: 66ch.
Make reading easier 🤘

**Review by msfernandes — CHANGES_REQUESTED** [review]
When there's no opinion card the opinion onboarding is still opening and the text selection onboarding does not open. Is this correct?

**Comment by cfviotti** [comment]
Please check if everything is on point now!

File: src/static/js/components/opinionOnboarding/view.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  var self = this;
+
+  var cookie = getCookie(this.cookieName);
+  if (self.opinionOnboardingElement.data('isAuthenticated')) {
+    if (!cookie) {
+      events.showOpinionOnboarding.publish();
+    } else {
+      if (self.opinionModalView.hasActiveOpinionCards()) {
+        events.openOpinionModal.publish(null);
+      }
```
Comment (msfernandes) [inline src/static/js/components/opinionOnboarding/view.js:24]: If there is no opinion cards, should not trigger `events.showTextSelectionOnboarding`?

File: src/static/js/components/opinionOnboarding/view.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  this.publishers();
+};
+
+OpinionOnboardingView.prototype.publishers = function() {
+  var self = this;
+
+  var cookie = getCookie(this.cookieName);
+  if (self.opinionOnboardingElement.data('isAuthenticated')) {
+    if (!cookie) {
+      events.showOpinionOnboarding.publish();
```
Comment (msfernandes) [inline src/static/js/components/opinionOnboarding/view.js:20]: Isn't better publish this event only if there is opinionCards?

**Review by msfernandes — CHANGES_REQUESTED** [review]

**Pull request merged**